packages/terra-site/src/examples/card/CardDefault.jsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import Card from 'terra-card';
+
+const CardDefault = () => (
+  <div>
```

Comment:
since our card background is white, and we don't have any padding or spacing on this one, you may want to throw some padding on this div and add a background color to make the example a bit more realistic

![screen shot 2017-08-15 at 10 42 50 am](https://user-images.githubusercontent.com/8029578/29323439-84ba4312-81a6-11e7-8736-2fb5a8914ef0.png)
vs

![screen shot 2017-08-15 at 10 41 53 am](https://user-images.githubusercontent.com/8029578/29323446-8b5f19e0-81a6-11e7-8f8b-ceb1b742a519.png)
I would check with UX or the core terra team though to make sure whatever we do aligns with their upcoming goals to style the site up.